src/index.html

```diff
@@ -11,6 +11,44 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <!-- <h1>Search bar airbnb</h1> -->
+    <form 
+      action="http://localhost:8080/" 
+      method="post"
+      data-qa="big">
+      <div class="big-icon">
+        <img 
+          class="pic" 
+          src="images/Search.svg"
+          alt="search-icon"
+          >
+        </img>
+        <input
+          data-qa="keypress"
+          class="bar big-bar"  
+          type="text" 
+          name="searchBar"
+          placeholder="Try &quot;Los Angeles&quot;"
+          >
+        </input>
+      </div>
+    </form>
+    <form 
+      action="http://localhost:8080/" 
```

Comment:
```suggestion
      action="#" 
```

Reply:
you can # as mock url, so it will act as an anchor and scroll to the top of the page